File: Partify.Domain/Entities/Customer.cs

using Partify.Domain.Entities.Base;

namespace Partify.Domain.Entities
{
    public  class Customer : AuditableEntity
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string? Email { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Address { get; set; }
        public string? CarModel { get; set; }
        public string? PlateNumber { get; set; }
    }
}

File: partify.react/src/types/Customer.ts
export interface Customer {
    id: number;
    name?: string;
    lastName?: string;
    email?: string;
    phoneNumber?: string;
    address?: string;
    carModel?: string;
    plateNumber?: string;
}

export interface CustomerAdd {
    name: string;
    lastName: string;
    email?: string;
    phoneNumber?: string;
    address?: string;
    carModel?: string;
    plateNumber?: string;
}

export interface CustomerUpdate {
    name?: string;
    lastName?: string;
    email?: string;
    phone?: string;
    address?: string;
    carModel?: string;
    plateNumber?: string;
}